rust_build_utils/rust_utils.py

```diff
@@ -129,7 +134,7 @@ def config_local_env_vars(config, local_config):
             if not "env" in GLOBAL_CONFIG[config.target_os]:
                 GLOBAL_CONFIG[config.target_os]["env"] = {env: tuple}
                 return
```

Comment:
also this is not related, but I suspect this return is a bug. Would you be willing to write UTs for this functions since you're doing fixes here?